sass-transformer.js

```diff
@@ -67,6 +67,7 @@ if ( reactNativeMinorVersion >= 59 ) {
 const autoImportIncludePaths = [
 	path.join( path.dirname( __filename ), 'src' ),
 	path.join( path.dirname( __filename ), 'gutenberg/assets/stylesheets' ),
```

Comment:
This line can be removed.
```suggestion
```

Reply:
It can, after https://github.com/WordPress/gutenberg/pull/17883 is merged 😉 
I left that so we don't have to coordinate merging both PRs. Once the base-styles are merged, we can remove that line in another PR.

Reply:
I think it's a good strategy in this case to keep both the "old" and the new paths so we smoothly transition to the new one. I'm approving this PR, but let us know if you have some more feedback here @gziolo and we can address it in the follow up PR.